config/config-policy-controller.yaml

```diff
@@ -18,4 +18,4 @@ metadata:
   name: config-policy-controller
   namespace: cosign-system
 data:
-  no-match-policy: allow
+  no-match-policy: deny
```

Comment:
If this is the default, then I'd recommend that we omit it entirely.

By specifying this, we make it so that folks upgrading using our configs will overwrite any changes they make to these.

Reply:
But this configmap hasn't been released yet. I don't see any harm on doing this change to the config file.

Reply:
Okay, i understood the intention now.